src/test/java/BaseTest.java

```diff
@@ -1,11 +1,30 @@
+<<<<<<< Updated upstream
```

Comment:
Its seen like the branch has conflic, please check and remove some line invalid 
-> remove <<<<<<< Updated upstream